### PR TITLE
Support bufferline

### PR DESCRIPTION
**Pull request by Isrothy**
Support [bufferline](https://github.com/akinsho/bufferline.nvim)

### CODE REVIEW

**Comment by Isrothy** [comment]
Also support notify

**Comment by Isrothy** [comment]
support [nvim-ts-rainbow](https://github.com/p00f/nvim-ts-rainbow)

image

**Comment by Isrothy** [comment]
NeoTree support
image

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 doc/tags
+.luarc.json
```
Comment (gbprod) [inline .gitignore:2]: Can you remove this please 😉

File: lua/nord/colors.lua [hunk 2]
```diff
@@ -24,5 +24,21 @@ return {
...
+  nord5 = "#E5E9f0",
+  nord6 = "#EcEFF4",
+  nord7 = "#8FBCBB",
+  nord8 = "#88C0D0",
+  nord9 = "#81A1C1",
+  nord10 = "#5E81AC",
+  nord11 = "#BF616A",
+  nord12 = "#D08770",
+  nord13 = "#EBCB8B",
+  nord14 = "#A3BE8C",
```
Comment (gbprod) [inline lua/nord/colors.lua:41]: Please use already defined color and do not duplicate.

**Review by gbprod — CHANGES_REQUESTED** [review]
Just a quick review for the moment, I'm not sure about your bufferline integration. I'll review this more tomorrow

**Comment by Isrothy** [comment]
Done

**Comment by gbprod** [comment]
Thanks for your work!

Please, next time, create 1 PR by plugin. 
I have a lot of little changes to do, I think it will be easier if I do it by myself. 
I'm working on a way to customize this theme more clearly, so, the bufferline api will change.

I think I'll merge this PR in a separate branch and apply my changes, I'll notify you for reviews.

**Comment by Isrothy** [comment]
Thanks a lot for reviewing!

May I ask what do you mean by "create 1 PR by plugin"? Does it mean plugins like [octokit-plugin-create-pull-request](https://github.com/gr2m/octokit-plugin-create-pull-request)? (just googled it)
Thanks for your time.

**Pull request merged**